src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -38,7 +38,9 @@ export default function * loadCityContent (
 
   if (!criterion.peeking()) {
     const appSettings = new AppSettings()
-    yield call(() => NotificationsManager.subscribeNews(newCity, newLanguage))
+    if (buildConfig().featureFlags.pushNotifications) {
```

Comment:
I think it would be cleaner to handle this check within the NotificationsManager.
That way you can always call subscribeNews but it will have no effect if PushNotifications are disabled. 

Maybe NotificationsManager should be renamed to PushNotificationsManager or similar.